### PR TITLE
MTM-49613 Updated screenshots and docs for getting started guide

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

**Review by reey — APPROVED** [review]

File: content/users-guide/getting-started-bundle/gui-features.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The general structure common to all {{< product-c8y-iot >}} applications include
 </tr>
 <tr>
 <td style="text-align:left"><b>Right drawer</b></td>
-<td style="text-align:left">Clicking the small arrow at the very right of the top bar will unhide/hide the right drawer, offering quick links to other applications and to relevant documentation. Per default, the right drawer is hidden.</td>
+<td style="text-align:left">Clicking the user icon at the very right of the top bar will unhide/hide the right drawer, offering quick links to other applications and to relevant documentation. Per default, the right drawer is hidden.</td>
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/gui-features.md:50]:
```suggestion
<td style="text-align:left">Clicking the user icon at the very right of the top bar will unhide/hide the right drawer, offering access to the user settings, quick links to other applications and to relevant documentation. Per default, the right drawer is hidden.</td>
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 2]
```diff
@@ -69,10 +69,10 @@ Several account settings can be changed by the user.
 2. In the **Edit user** dialog box, make the relevant changes.
 3. To change the language, select a [language](#languages) from the dropdown list in the  **Language** field.
 4. Use the **Product experience** toggle to opt-out from the product experience tracking and the Knowledge Hub by Gainsight which are both enabled by default if you accepted functional cookies.
-5. Click **Change password** to change your current password, see [To change your password](#change-password) for details.
-6. Click **Save** to apply your settings.
-
-<img src="/images/users-guide/getting-started/getting-started-user-settings.png" alt="User settings"  style="max-width: 100%">
+5. Use the **In-product information** toggle to opt-out from the information provided within the product.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:72]: from the information provided within the product > what exactly does that mean/include?

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 3]
```diff
@@ -69,10 +69,10 @@ Several account settings can be changed by the user.
...
+6. Click **Change password** to change your current password, see [To change your password](#change-password) for details.
+7. Click **Set up two-factor authentication** to set up two-factor authentication and increase tenant security.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:74]: Does this show up for a normal user?

Comment (eniosultan) [reply content/users-guide/getting-started-bundle/user-settings.md:74]: I tested it with a user without admin persmissions and it was still available.

File: content/users-guide/getting-started-bundle/user-settings.md (hunk 2 above)
Comment (eniosultan) [reply content/users-guide/getting-started-bundle/user-settings.md]: fixed explanation

**Comment by BeateRixen** [comment]
@eniosultan @reey I assume the new right drawer and the move of the user menu will be described in a separate PR? I have not yet seen any changes for that.

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**